modules/images-configuration-insecure.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
 // * openshift_images/image-configuration.adoc
+// * post_installation_configuration/preparing-for-users.adoc
 
 [id="images-configuration-insecure_{context}"]
 = Importing insecure registries and blocking registries
```

Comment:
The format of registries.conf  should be as below:
$ oc debug nodes/<Node_name>
Starting pod/compute-0-debug ...
To use host binaries, run `chroot /host`
sh-4.2# chroot /host
sh-4.4# cat /etc/containers/registries.conf 
unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]

[[registry]]
  prefix = ""
  location = "docker.io"
  blocked = true
[[registry]]
  prefix = ""
  location = "insecure.com"
  insecure = true

Reply:
I think another change is needed for the overall doc:
"blacklist" with "denylist"
"whitelist" to "allowlist"